src/pages/api/properties.js

```diff
@@ -0,0 +1,12 @@
+import { getProperties } from '../../utils/service-api-client/properties'
+
+export default async (req, res) => {
+  try {
+    const { status, data } = await getProperties(req.query)
+
+    res.status(status).json(data)
+  } catch (err) {
+    console.log(err)
+    return { status: 500 }
+  }
+}
```

Comment:
Do you think this should be `pages/api/properties/index.js`?

Reply:
I see what you mean. We could do. The docs don't seem to recommend one way over another. https://nextjs.org/docs/api-routes/dynamic-api-routes#index-routes-and-dynamic-api-routes